DiscordPy/bot.py

```diff
@@ -70,11 +72,12 @@ async def getHelp(ctx, *arg):
 
 @bot.command(name='meme')
 async def getMeme(ctx, *arg):
-    if (len(imageList) == 0):
-        hot_python = subreddit.new(limit=20)
+    delta_time = datetime.datetime.now() - prev_time
+    delta_seconds = delta_time.total_seconds()
+    if (len(imageList) == 0 or delta_seconds >= 60):
+        hot_python = subreddit.new(limit=100)
         for i in hot_python:
-            extension = i.url[-3:]
-            if (extension == "jpg"):
+            if i.link_flair_text in {"Meme", "Loss", "Shitpost"} and i.url[-3:] in {"jpg", "png"}:
                 imageList.append(i.url)

```

Comment:
I need to remove all the imageURL from the list if I am to append them. What I am doing will be re-populating old imageURL which I do not want.